extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -246,16 +246,16 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 
 		// Validate Rows Filegroup names
 		if (this.objectInfo.filegroups?.length > 0) {
-			let seenFilegroups = new Set(this.objectInfo.filegroups.map(function (item) { return item.name }));
-			if (seenFilegroups.size !== this.objectInfo.filegroups.length) {
-				this.objectInfo.filegroups.forEach((item) => {
-					if (seenFilegroups.has(item.name)) {
-						seenFilegroups.delete(item.name);
-					} else {
-						errors.push(localizedConstants.FilegroupExistsError(item.name));
-					}
-				});
-			}
+			let seenFilegroups = new Set<string>;
+			this.objectInfo.filegroups.map(function (item) {
+				if (item.name === '') {
```

Comment:
Can the name ever be undefined here?

Reply:
Actually No, to your question, As a new row always initilaizes with name: ' '. we dont get undefined here!